src/screens/PaymentDetailsScreen.tsx

```
@@ -1,29 +1,98 @@
-      <h1>Payment Details</h1>
-  <h2>{payment.title}</h2>
-  <p>Shares:</p>
-  <ul>
-  {Array.from(payment.shares.entries()).map(([name, amount]) => (
-      <li key={name}>{name}: {amount}</li>
-))}
-  </ul>
-  </div>
-);
+      <div className="field">
+        <label className="label">Payment Title</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)